theme/src/components/dark-text-input.js

```diff
@@ -16,5 +16,11 @@ const DarkTextInput = styled(TextInput)`
     outline: none;
     box-shadow: none;
   }
+
+  &::-webkit-input-placeholder {
+    // Needed to fix Chromium-based Edge
+    // where user-agent color is too dark.
+    color: rgb(117, 117, 117);
```

Comment:
Should we use a color from the theme here?

Reply:
I was keeping consistency with the user-agent. If we want to switch it to something, I'd say we move to what primer/css uses, which is:
```
&::placeholder {
    color: inherit;
    opacity: 0.6; // inceases contrast ratio to 4.52
  }
```

Reply:
☝️ I'm good with that